exercises/practice/list-ops/.meta/proof.ci.wren

```diff
@@ -0,0 +1,104 @@
+class ListOps {
+  construct new() {
+    _data = []
+  }
+
+  construct new(elements) {
+    _data = elements[0..-1]
+  }
+
+  toList { _data[0..-1] }
+
+  add(element) {
+    _data.add(element)
+  }
+
+  // https://wren.io/control-flow.html#the-iterator-protocol
+  iterate(iterator) {
+    // first iteration, wren provides the null value
```

Comment:
Added a touch of commentary for those reading the proof who are unfamiliar with the iterator protocol